op-acceptance-tests/tests/interop/loadtest/invalid_msg_test.go

```diff
@@ -0,0 +1,121 @@
+package loadtest
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/txinclude"
+	"github.com/ethereum-optimism/optimism/op-service/txintent"
+	suptypes "github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
+	"github.com/ethereum/go-ethereum/core"
+)
+
+type makeInvalidInitMsgFn func(suptypes.Message) suptypes.Message
+
+func makeInvalidBlockNumber(msg suptypes.Message) suptypes.Message {
+	msg.Identifier.BlockNumber++
+	return msg
+}
+
+// InvalidExecutor spams invalid executing messages, aiming to stress mempool interop filters.
+type InvalidExecutor struct {
+	l2             *L2
+	eoa            *SyncEOA
+	validInitMsg   suptypes.Message
+	makeInvalidFns *RoundRobin[makeInvalidInitMsgFn]
+}
+
+var _ Spammer = (*InvalidExecutor)(nil)
+
+// NewInvalidExecutor returns an InvalidExecutor. It assumes  validInitMsg is a valid initiating
+// message on a source chain.
+func NewInvalidExecutor(l2 *L2, validInitMsg suptypes.Message) *InvalidExecutor {
+	// Fund an EOA that will be spamming the invalid transactions. It should never need to spend
+	// any wei, but we don't want to trigger mempool balance checks.
+	eoa := l2.Funder.NewFundedEOA(eth.OneEther)
+
+	// The InvalidExecutor uses an txinclude.Includer to manage nonces concurrently. It uses a
+	// txinclude.Sender implementation that fails on the first error. Because we anticipate
+	// failure, this enables us to spam without entering infinite resubmission loops.
+	unreliableSender := struct {
```

Comment:
I'm curious how this methodless struct instance is being used as a parameter to NewPersistent() below, unless it's type in NewPersistent() is any.